problems/arraysWithLoops.js

```diff
@@ -66,15 +122,38 @@ function findSmallest() {}
  * @returns {number} second smallest value
  */
 
-function findSecondSmallest() {}
+function findSecondSmallest(nums) {
+  let smallestNum = nums[0]
+  let secondSmallest = nums[1]
+  for (let i = 1; i < nums.length; i+= 1){
+    if (nums[i] < smallestNum ){
+      secondSmallest = smallestNum
+      smallestNum = nums[i]
+      } else if (nums[i]< secondSmallest) {
+        secondSmallest = nums[i]
+      }
+    } return secondSmallest
+}
 
 /**
  * Takes in array and returns the second largest number in the array
  * @param {number[]} nums
  * @returns {number} second largest value
  */
 
-function findSecondLargest() {}
+function findSecondLargest(nums) {
+  let max = nums[0] > nums[1] ? nums[0] : nums[1]
+  let secondMax = nums[0] > nums[1] ? nums[1] : nums[0]
```

Comment:
you just set these to be equal to negative Inifinity to avoid having to deal with checking which is larger